### DISCUSSION TITLE
Notifications page 500 when clicking on the "All" filter - https://github.com/notifications

### DISCUSSION THREAD

**Discussion by johnmarkredding** [community/community#77370] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

MacBook Air 11-inch Early 2014
MacOS `11.7.10` `(20G1427)`
Chrome browser | `Version 119.0.6045.159 (Official Build) (x86_64)`

Notifications page error: https://github.com/notifications

I run into a 500 error when clicking on the `All` filter button, but only when this option is already active.

It takes me to a new page at this url: https://github.com/notifications/beta/set_preferred_inbox_query

This video shows everything you need, I believe.

https://github.com/community/community/assets/15619295/56cb8c90-658e-4ff3-90f4-4abf61aa91d1

Here is what I see in my browser console when I click on the filter.
![Console visible error](https://github.com/community/community/assets/15619295/b8fcbc03-a9dd-4efe-8d76-d0475c7fe0ca)

**Comment by DNin01** [comment]
Can replicate. Only happens if I click "All" when it is already selected, as you have noted. Not a huge deal, though.

**Reply by johnmarkredding** [reply]
Nope, just thought I'd share. Hope it helps.

**Comment by johnmarkredding** [comment, marked as answer]
I can't reproduce the issue today, so I assume it's fixed.
